http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala

```diff
@@ -250,6 +253,89 @@ object RuntimeRoutes {
       c.getOrElse(Map.empty)
     )
   }
+
+  //TODO: add `dataprocInstances` when it's added to `RuntimeConfig` https://broadworkbench.atlassian.net/browse/IA-1778
```

Comment:
Agreed `dataprocInstances` makes sense in `RuntimeConfig`. Will need to do a DB migration.

Reply:
As far as I know clients don't use this field, so should be fine to leave it out for now.